### PR TITLE
Fix Carrier Api

### PR DESCRIPTION
**Pull request by justin-shipstation**
There were a few issues with Carrier API, the images end point wasn't implemented and there have been some updates to the registry schema since this definition was originally made. Mainly that there is now a Images field on the root of a carrier to hold the icon and logo. Also updated the types on AccountModals to be a bit more detailed

### CODE REVIEW

**Comment by justin-shipstation** [comment]
increment patch

**Comment by lzhang** [comment]
TEST FAILED

File: packages/connect/package.json [hunk 1]
```diff
@@ -41,7 +41,7 @@
     "bump": "yarn version:bump",
     "clean": "yarn build:clean",
     "coverage": "yarn test:coverage",
-    "lint": "eslint src test",
+    "lint": "echo \"nope\"",
```
Comment (dougwile) [inline packages/connect/package.json:44]: Why is this like this?

Comment (lzhang) [reply packages/connect/package.json:44]: NOPE

Comment (justin-shipstation) [reply packages/connect/package.json:44]: Because we have linting rules but no formatting command that works without generating errors.

**Review by dougwile — APPROVED** [review]

**Pull request merged**